### DISCUSSION TITLE
Firefly Importer fails mapping

### DISCUSSION THREAD

**Discussion by quotary** [firefly-iii/firefly-iii#8367] [Firefly III] [answered]
When trying to import a CAMT.053 file and submitting the job after checking and mapping data (which it reads/shows correctly in the Role definition screen) it fails instantly with the following error message in the logs:

> firefly_iii_importer  | [2024-01-09 14:18:17] local.DEBUG: [1/9] Now mapping.
firefly_iii_importer  | [2024-01-09 14:18:17] local.DEBUG: Transaction has 1 split(s)
firefly_iii_importer  | [2024-01-09 14:18:17] local.DEBUG: Description is "58823456 1030990830899255 58823456 - websiteX.com"
firefly_iii_importer  | [2024-01-09 14:18:17] local.ERROR: abs(): Argument #1 ($num) must be of type int|float, null given {"exception":"[object] (TypeError(code: 0): abs(): Argument #1 ($num) must be of type int|float, null given at /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php:522)
firefly_iii_importer  | [stacktrace]
firefly_iii_importer  | #0 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(522): abs()
firefly_iii_importer  | #1 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(372): App\\Services\\Camt\\Conversion\\TransactionMapper->mapTransactionJournal()
firefly_iii_importer  | #2 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(49): App\\Services\\Camt\\Conversion\\TransactionMapper->mapTransactionGroup()
firefly_iii_importer  | #3 /var/www/html/app/Services/Camt/Conversion/RoutineManager.php(133): App\\Services\\Camt\\Conversion\\TransactionMapper->map()
firefly_iii_importer  | #4 /var/www/html/app/Http/Controllers/Import/ConversionController.php(194): App\\Services\\Camt\\Conversion\\RoutineManager->start()
firefly_iii_importer  | #5 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Http\\Controllers\\Import\\ConversionController->start()
firefly_iii_importer  | #6 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(43): Illuminate\\Routing\\Controller->callAction()
firefly_iii_importer  | #7 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(259): Illuminate\\Routing\\ControllerDispatcher->dispatch()
firefly_iii_importer  | #8 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
firefly_iii_importer  | #9 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(806): Illuminate\\Routing\\Route->run()
firefly_iii_importer  | #10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
firefly_iii_importer  | #11 /var/www/html/app/Http/Middleware/IsReadyForStep.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #12 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): App\\Http\\Middleware\\ConversionControllerMiddleware->handle()
firefly_iii_importer  | #13 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
firefly_iii_importer  | #15 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #16 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
firefly_iii_importer  | #17 /var/www/html/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #18 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
firefly_iii_importer  | #19 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #20 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
firefly_iii_importer  | #21 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\StartSession->handle()
firefly_iii_importer  | #22 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #23 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
firefly_iii_importer  | #24 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #25 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
firefly_iii_importer  | #26 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #27 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(805): Illuminate\\Pipeline\\Pipeline->then()
firefly_iii_importer  | #28 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(784): Illuminate\\Routing\\Router->runRouteWithinStack()
firefly_iii_importer  | #29 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(748): Illuminate\\Routing\\Router->runRoute()
firefly_iii_importer  | #30 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(737): Illuminate\\Routing\\Router->dispatchToRoute()
firefly_iii_importer  | #31 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\\Routing\\Router->dispatch()
firefly_iii_importer  | #32 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
firefly_iii_importer  | #33 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #34 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
firefly_iii_importer  | #35 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
firefly_iii_importer  | #36 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #37 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
firefly_iii_importer  | #38 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
firefly_iii_importer  | #39 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #40 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
firefly_iii_importer  | #41 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(99): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #42 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
firefly_iii_importer  | #43 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #44 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\HandleCors->handle()
firefly_iii_importer  | #45 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #46 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle()
firefly_iii_importer  | #47 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
firefly_iii_importer  | #48 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\\Pipeline\\Pipeline->then()
firefly_iii_importer  | #49 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
firefly_iii_importer  | #50 /var/www/html/public/index.php(76): Illuminate\\Foundation\\Http\\Kernel->handle()
firefly_iii_importer  | #51 {main}
firefly_iii_importer  | "}

Resubmitting the job does trigger the import, but stays in a loop. I've waited for >30 minutes for an import of just 9 lines, thus stopped it. It keeps showing the following lines over and over: 
> firefly_iii_importer  | 192.168.0.X- - [09/Jan/2024:14:18:17 +0100] "POST /import/convert/start?identifier=conv-X HTTP/1.1" 500 1144 "http://192.168.0.X:81/import/convert" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0"
firefly_iii_importer  | [2024-01-09 14:18:18] local.DEBUG: isReadyForFileStep("conversion")
firefly_iii_importer  | [2024-01-09 14:18:18] local.DEBUG: isReadyForFileStep: Return true
firefly_iii_importer  | [2024-01-09 14:18:18] local.DEBUG: Now at App\Http\Controllers\Import\ConversionController::status(conv-X)
firefly_iii_importer  | [2024-01-09 14:18:18] local.DEBUG: Now in startOrFindConversion(conv-X)
firefly_iii_importer  | [2024-01-09 14:18:18] local.DEBUG: Conversion status is "conv_running"
Might be of useful information for a check and potential fix so this doesn't happen?

I'm using the docker compose installation where the VANITY_URL in .env goes to the local IP instead of localhost.

Please let me know if more information is needed, like a snippet of the CAMT.053 file to be imported, censored.

**Comment by JC5** [comment]
No that looks like a bug, thanks for letting me know! I'll create an issue about it later today.

**Reply by quotary** [reply]
Thanks for the quick reply! I look forward to the potential fix so I can import my transactions instead of creating them by hand. :)

**Reply by quotary** [reply]
Quick update: I have updated to Firefly importer 1.4.1 and the issue still persists with the same errors. Also tried with another CAMT file which also fails.

---

Debug information generated at 2024-01-16 21:49:49 Europe/Amsterdam for Firefly III Data Importer version **1.4.1**.

<table><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>Version</td><td>1.4.1</td></tr><tr><td>Build</td><td><span>#</span>, base <span>#</span>78</td></tr><tr><td>System</td><td>PHP 8.3.1, Linux, apache2handler</td></tr></table><table><tr><th colspan="2">App information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>Timezone</td><td>Europe/Amsterdam, Europe/Amsterdam</td></tr><tr><td>Environment</td><td>local</td></tr><tr><td>Debug mode</td><td>false, cache 'file'</td></tr><tr><td>Log level</td><td>debug, stack</td></tr><tr><td>Display errors</td><td>Off, ALL errors</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Trusted proxies</td><td></td></tr></table><table><tr><th colspan="2">User information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0</td></tr></table>


```
Truncated from this point <----|of type int|float, null given at /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php:516)
[stacktrace]
#0 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(516): abs()
#1 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(346): App\\Services\\Camt\\Conversion\\TransactionMapper->mapTransactionJournal()
#2 /var/www/html/app/Services/Camt/Conversion/TransactionMapper.php(45): App\\Services\\Camt\\Conversion\\TransactionMapper->mapTransactionGroup()
#3 /var/www/html/app/Services/Camt/Conversion/RoutineManager.php(117): App\\Services\\Camt\\Conversion\\TransactionMapper->map()
#4 /var/www/html/app/Http/Controllers/Import/ConversionController.php(183): App\\Services\\Camt\\Conversion\\RoutineManager->start()
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Http\\Controllers\\Import\\ConversionController->start()
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(43): Illuminate\\Routing\\Controller->callAction()
#7 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(259): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#8 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(205): Illuminate\\Routing\\Route->runController()
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(806): Illuminate\\Routing\\Route->run()
#10 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(144): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#11 /var/www/html/app/Http/Middleware/IsReadyForStep.php(44): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): App\\Http\\Middleware\\ConversionControllerMiddleware->handle()
#13 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#14 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
#15 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#16 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#17 /var/www/html/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#18 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#19 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#20 /var/www/html/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#21 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Session\\Middleware\\StartSession->handle()
#22 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#23 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#24 /var/www/html/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#25 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#26 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#27 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(805): Illuminate\\Pipeline\\Pipeline->then()
#28 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(784): Illuminate\\Routing\\Router->runRouteWithinStack()
#29 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(748): Illuminate\\Routing\\Router->runRoute()
#30 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(737): Illuminate\\Routing\\Router->dispatchToRoute()
#31 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(200): Illuminate\\Routing\\Router->dispatch()
#32 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(144): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#33 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#34 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#35 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#36 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#37 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#38 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#39 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#40 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#41 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(99): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#42 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#43 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#44 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Http\\Middleware\\HandleCors->handle()
#45 /var/www/html/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#46 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(183): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#47 /var/www/html/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(119): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#48 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(175): Illuminate\\Pipeline\\Pipeline->then()
#49 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(144): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#50 /var/www/html/public/index.php(76): Illuminate\\Foundation\\Http\\Kernel->handle()
#51 {main}
"} 
[2024-01-16 21:49:22] local.DEBUG: isReadyForFileStep("conversion")  
[2024-01-16 21:49:23] local.DEBUG: isReadyForFileStep: Return true  
[2024-01-16 21:49:23] local.DEBUG: Now at App\Http\Controllers\Import\ConversionController::status(conversion-3S6G2Ydfw28i)  
[2024-01-16 21:49:23] local.DEBUG: Now in startOrFindConversion(conversion-3S6G2Ydfw28i)  
[2024-01-16 21:49:23] local.DEBUG: Conversion status is "conv_running"  

```

**Reply by JC5** [reply]
Mmm, that's strange. Could you send me the file you're trying to import at james@firefly-iii.org? I'll have to do some debugging.

**Reply by JC5** [reply]
Alternatively, try running `:develop`, it should be fixed there.

**Reply by quotary** [reply]
I have tried running `:develop`, problem still persists however. To confirm the lines in docker-compose.yml are as follows:
```  
importer:
    image: fireflyiii/data-importer:develop
```

I will send you the file via email, you can expect it in a couple of minutes. :)

**Reply by JC5** [reply]
Should be fixed in the latest `:develop`, thanks for helping me pinpoint the issue!

**Comment by quotary** [comment, marked as answer]
I can confirm this has been fixed in the latest `:develop`. Thanks for fixing!
